mockwebserver/src/test/java/mockwebserver3/MockWebServerTest.kt

```diff
@@ -71,11 +71,10 @@ class MockWebServerTest {
   @RegisterExtension
   var platform = PlatformRule()
 
-  private lateinit var server: MockWebServer
+  private val server = MockWebServer()
```

Comment:
This pattern now feels unsafe, since `server.stop()` must be called.

But existing problem.